pkg/util/iptables/iptables.go

```diff
@@ -361,7 +363,7 @@ func (runner *runner) checkRuleWithoutCheck(table Table, chain Chain, args ...st
 }
 
 func unquote(strp *string) {
-	if len(*strp) >= 2 && (*strp)[0] == '"' && (*strp)[len(*strp)-1] == '"' {
+	if len(*strp) >= 2 && ((*strp)[0] == '"' || (*strp)[len(*strp)-1] == '"') {
```

Comment:
This allows for args like `"foo` and `bar"` - under what conditions does this arise?